doc/source/examples/01_getting_started/05_plotter_picker.mystnb

```diff
@@ -179,6 +179,30 @@ plotter = PlotterHelper(use_trame=True)
 plotter.plot(plot_list)
 ```
 
+## Clipping objects
```

Comment:
```suggestion
## Clip objects
```
Change to simple verb tense (rather than gerund -ing form) to match other sections